code/modules/mob/living/carbon/xenomorph/hive_datum.dm

```diff
@@ -928,6 +946,12 @@ to_chat will check for valid clients itself already so no need to double check f
 
 	orphan_hud_timer = new(null, null, get_all_xenos(), D.orphan_hive_timer, "Orphan Hivemind Collapse: ${timer}", 150, -80)
 
+//This creates the hud timer for siloless collapse.
+/datum/hive_status/normal/proc/setup_siloless_hud_timer()
+	SIGNAL_HANDLER
+	var/datum/game_mode/infestation/nuclear_war/D = SSticker.mode
+	siloless_hud_timer = new(null, null, get_all_xenos() , D.siloless_hive_timer, "Siloless Collapse: ${timer}")
```

Comment:
for the record that check needs to be everywhere where NW specific stuff is used, which is why I said it would be fine just having it be empty on gamemode